src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -1,8 +1,17 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 import * as path from "path";
-import * as semver from "semver";
-import vscode, { Extension, extensions, Uri } from "vscode";
+import vscode, { Uri } from "vscode";
+import { AccessToken } from "@azure/identity";
+import { AzLoginTreeItem } from "../tree/AzLoginTreeItem";
+import { AzManagementGroupTreeItem } from "../tree/AzManagementGroupTreeItem";
+import { AzResourceGroupTreeItem } from "../tree/AzResourceGroupTreeItem";
```

Comment:
@shenglol Frankly I'm not convinced this is worth the effort.  What advantage does it give?

Reply:
It's a way to simplify the imports, so instead of doing
```
import { X } from "foo/x";
import { Y } from "foo/y;
import { Z } from "foo/z;
```
we can just do
```
import { X, Y, Z } from "foo";
```
I don't have a strong opinion on this but thought this can save a few of lines.